packages/core/src/domain/configuration.ts

```diff
@@ -13,6 +13,7 @@ export const DEFAULT_CONFIGURATION = {
   silentMultipleInit: false,
   trackInteractions: false,
   trackViewsManually: false,
+  censorshipLevel: 'PUBLIC',
```

Comment:
* It could be interesting to pass recording options to the `startSessionReplayRecording` function instead of `init`, so they are scoped and can be changed every time the recording is restarted
* I'm not found of `censorship`: I think it is a bit negatively connoted. What about something like `privacyLevel` or `maskMode`?
* We should have an object constants (ex: similar to [`StatusType`](https://github.com/DataDog/browser-sdk/blob/5331c1f33324458e983a4c8a689b53f7131c1bde/packages/logs/src/domain/logger.ts#L12-L20)) to avoid hardcoding values

Reply:
`startSessionReplayRecording` is interesting. 🤔